app/controllers/admin/conference_controller.rb

```diff
@@ -174,8 +174,27 @@ def edit
       end
     end
 
+    def conference_wide_screen
+      # To display sponsors in the conference wide information page
+      @conference = Conference.find_by(short_title: params[:id])
```

Comment:
This shouldn't be necessary

``
 load_and_authorize_resource :conference, find_by: :short_title
``
already loads @conference. 

https://github.com/CanCanCommunity/cancancan/wiki/Authorizing-controller-actions